### DISCUSSION TITLE
word/2023-ZEXAL

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#425] [Announcements]
# word/2023-ZEXAL

充满跌宕的2022年已过，2023年也得继续加把劲 ✊✊✊ 新的一年，也要一飞冲天！ がとビングだぜ、俺！ ZEXAL大好き❤️

http://101.43.83.239:8028/word/2023-ZEXAL.html

**Comment by StarNextDay** [comment]
尝试下评论功能：）
